### PR TITLE
init: Boostrap project

### PR DESCRIPTION
**Pull request by camh-**
Bootstrap the project by installing the standard foxygoat tooling for Go
programs:

* Go 1.19
* golangci-lint
* hermit
* goreleaser
* make

Use the standard Makefile tool configs, copied from foxygoat/evy.

Apache license as usual.

### CODE REVIEW

File: bin/hermit.hcl [hunk 1]
```diff
@@ -0,0 +1,10 @@
+sources = [
+  "https://github.com/cashapp/hermit-packages.git",
+  "https://github.com/foxygoat/hermit-packages.git",
+]
+env = {
+  GOBIN: "${HERMIT_ENV}/out/bin",
+  PATH: "${GOBIN}:${PATH}",
+}
+manage-git = false
+
```
Comment (juliaogris) [inline bin/hermit.hcl]: intentional newline?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+API. With this control interface, we can turn the TV on and off.
+
+By hooking into the XSCREENSAVER extension, we can watch when the screen saver
+is enabled and turn off the TV. When the screen saver is disabled, we can turn
+the TV on. To be even smarter, we can use the RANDR extension to check if the TV
+is actually connected to the computer so that we do not try to turn it on and
+off if it is not actually plugged into the machine. This is handy for laptops
+which may get unplugged but remain on wifi, so are still able to control the TV
+when we may not want it to.
+
```
Comment (juliaogris) [inline README.md]: wanted nl?

File: .github/workflows/cicd.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
```
Comment (juliaogris) [inline .github/workflows/cicd.yaml]: why do you want all history for the CI job?

File: main.go [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+`
+
+type CLI struct {
+	Version kong.VersionFlag `short:"V" help:"Print program version"`
+}
+
+func main() {
+	// Set maxprocs to 1 - this is a simple program and we don't want
+	// more than one kernel thread for it, even on large boxes.
+	runtime.GOMAXPROCS(1)
```
Comment (juliaogris) [inline main.go:22]: this seems to indicate that go might be doing something different?
just more a curiosity. in general my feeling is less code is better unless there is a good reason.

**Review by juliaogris — APPROVED** [review]
lgtm

File: bin/hermit.hcl (hunk 1 above)
Comment (camh-) [reply bin/hermit.hcl]: no. removed.

File: README.md (hunk 2 above)
Comment (camh-) [reply README.md]: removed

File: .github/workflows/cicd.yaml (hunk 3 above)
Comment (camh-) [reply .github/workflows/cicd.yaml]: I copied it from the evy cicd.yaml. maybe firebase-deploy needed that, in which case i'll just remove it here.

File: main.go (hunk 4 above)
Comment (camh-) [reply main.go:22]: The default as of Go 1.18 is that it defaults to the number of cores/threads on the CPU. My desktop has 8 cores, and my next one may have 16. I recently read of some small programs ending up with a lot of threads when they really didn't need it. In this case, I really don't need it and I want to limit resource usage.

Comment (camh-) [reply main.go:22]: Sorry, Go 1.5 not 1.18. It was 2018, not 1.18

Comment (juliaogris) [reply main.go:22]: ok, cool thanks for the info!

**Pull request merged**